apis/core/lib/domain/dimension/DimensionMetadataCollection.ts

```diff
@@ -55,13 +62,23 @@ export default function Mixin<Base extends Constructor<Omit<DimensionMetadataCol
       return dimensionMetadata
     }
 
+    findDimension(params: FindDimensionMetadata): DimensionMetadata.DimensionMetadata | undefined {
+      const identifier = params.csvMapping.createIdentifier(params.targetProperty)
+
+      return this.find(identifier)
+    }
```

Comment:
I propose to merge this and `find` below
```suggestion
    find(params: FindDimensionMetadata | Term): DimensionMetadata.DimensionMetadata | undefined {
      const identifier = 'termType' in params ? params : params.csvMapping.createIdentifier(params.targetProperty)

      return this.hasPart.find(part => identifier.equals(part.about))
    }
```

Reply:
Done, 
@vhf I think this affects you